[2022-12-20T00:01:09.517+0100] {processor.py:153} INFO - Started process (PID=10978) to work on /home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-12-20T00:01:09.520+0100] {processor.py:743} INFO - Processing file /home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-12-20T00:01:09.522+0100] {logging_mixin.py:137} INFO - [2022-12-20T00:01:09.521+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-12-20T00:01:09.545+0100] {logging_mixin.py:137} INFO - [2022-12-20T00:01:09.544+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-12-20T00:01:09.545+0100] {logging_mixin.py:137} INFO - [2022-12-20T00:01:09.545+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-12-20T00:01:09.545+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-12-20T00:01:09.552+0100] {processor.py:175} INFO - Processing /home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.042 seconds
[2022-12-20T00:01:59.427+0100] {processor.py:153} INFO - Started process (PID=11062) to work on /home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-12-20T00:01:59.428+0100] {processor.py:743} INFO - Processing file /home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-12-20T00:01:59.429+0100] {logging_mixin.py:137} INFO - [2022-12-20T00:01:59.429+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-12-20T00:01:59.445+0100] {logging_mixin.py:137} INFO - [2022-12-20T00:01:59.445+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-12-20T00:01:59.445+0100] {logging_mixin.py:137} INFO - [2022-12-20T00:01:59.445+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-12-20T00:01:59.445+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-12-20T00:01:59.453+0100] {processor.py:175} INFO - Processing /home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2022-12-20T00:02:53.461+0100] {processor.py:153} INFO - Started process (PID=11200) to work on /home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-12-20T00:02:53.470+0100] {processor.py:743} INFO - Processing file /home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-12-20T00:02:53.472+0100] {logging_mixin.py:137} INFO - [2022-12-20T00:02:53.472+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-12-20T00:02:53.502+0100] {logging_mixin.py:137} INFO - [2022-12-20T00:02:53.502+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-12-20T00:02:53.502+0100] {logging_mixin.py:137} INFO - [2022-12-20T00:02:53.502+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-12-20T00:02:53.502+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-12-20T00:02:53.509+0100] {processor.py:175} INFO - Processing /home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.053 seconds
[2022-12-20T00:03:37.612+0100] {processor.py:153} INFO - Started process (PID=11249) to work on /home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-12-20T00:03:37.622+0100] {processor.py:743} INFO - Processing file /home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-12-20T00:03:37.625+0100] {logging_mixin.py:137} INFO - [2022-12-20T00:03:37.624+0100] {dagbag.py:538} INFO - Filling up the DagBag from /home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-12-20T00:03:37.654+0100] {logging_mixin.py:137} INFO - [2022-12-20T00:03:37.653+0100] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-12-20T00:03:37.654+0100] {logging_mixin.py:137} INFO - [2022-12-20T00:03:37.654+0100] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-12-20T00:03:37.654+0100] {processor.py:755} WARNING - No viable dags retrieved from /home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-12-20T00:03:37.661+0100] {processor.py:175} INFO - Processing /home/kamlesh/pyenvs/data-engg/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.057 seconds
